tools/topology/sof-hda-generic.m4

```diff
@@ -30,7 +30,7 @@ include(`platform/intel/intel-generic-dmic.m4')
 
 # The pipeline naming notation is pipe-PROCESSING-DIRECTION.m4
 # PPROC is set by makefile
-define(PIPE_VOLUME_PLAYBACK, `sof/pipe-`PPROC'-playback.m4')
+define(PIPE_HEADSET_PLAYBACK, `sof/pipe-`PPROC'-playback.m4')
```

Comment:
@xiulipan where is glk.m4? maybe all you need here is undefine before defining here?

Reply:
Sorry, it should be 
```
# Include bxt DSP configuration
include(`platform/intel/bxt.m4')
```
I will update the commit message.